src/atoms/CustomHtml/CustomHtml.js

```diff
@@ -1,34 +1,99 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
 import whitelist from '../../utils/whitelist.json';
 
 class CustomHtml extends React.Component {
   constructor(props) {
     super(props);
-
+    this.myRef = React.createRef();
     this.state = {
-      html: ''
+      html: '',
+      loaded: false
     };
   }
 
   componentDidMount() {
+    const { html, safeScripts } = this.parseHtml({
+      ...this.props.nodeData.attrs,
+      whitelist: whitelist.approved
+    });
+
+    // eslint-disable-next-line
+    const node = ReactDOM.findDOMNode(this.myRef.current);
+    if (!this.state.loaded) {
+      safeScripts.forEach((script) => {
+        this.loadScript(script.src, node);
```

Comment:
Does the setstate loaded need to be asynchronous or is this good enough?

Reply:
I think setState loaded might not even be needed because component should only mount once. I was having issues with functional components mounting the script multiple times when using useEffect. loaded state is set as false on the constructor and isn't updated until after these lines of code, so I don't think the asynchronous status would matter.

Reply:
ok did useEffect have the empty parens as the last param ?

Reply:
Would be great for this to use hooks but we can worry about that later